tests/test_work_display.py

```diff
@@ -329,3 +329,47 @@ def test_work_display_further_search_subject(self):
         )
         if not 'phrase.subject="venskab" and dkcclterm.dk="sk"' in new_input.get_attribute('value'):
             assert False
+
```

Comment:
Bare for at skrive noget, Der må gerne være en kommentar ved funktionen som fortæller hvad der testes ;-)